cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -0,0 +1,517 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+
+from aws_cdk import aws_cloudwatch as cloudwatch
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import core
+from aws_cdk.aws_cloudwatch import GraphWidget, LogQueryWidget, TextWidget
```

Comment:
You're already importing cloudwatch in the first line:
`from aws_cdk import aws_cloudwatch as cloudwatch`

so you can remove this extra import and in the code you can use cloudwatch.GraphWidget, cloudwatch.LogQueryWidget, cloudwatch.TextWidget

We're leaving the cdk module as prefix to be more clear when we're using a construct from CDK and one created by us.
cloudwath